pkg/config/default.go

```diff
@@ -55,7 +55,7 @@ var (
 			ListMergeStrategy: "replace",
 			Terminal: schema.Terminal{
 				MaxWidth: templates.GetTerminalWidth(),
-				Pager:    true,
+				Pager:    "less",
```

Comment:
Do we have a test confirming this works with arguments to less?

Reply:
added one